pkg/datapath/linux/devices.go

```diff
@@ -403,16 +403,6 @@ func (dm *DeviceManager) Listen(ctx context.Context) (chan []string, error) {
 	return devicesChan, nil
 }
 
-func (dm *DeviceManager) AreDevicesRequired() bool {
-	if option.Config.DryMode {
-		return false
-	}
-
-	return option.Config.EnableNodePort ||
-		option.Config.EnableHostFirewall ||
-		option.Config.EnableBandwidthManager
-}
-
```

Comment:
Is it safe to drop the check for `option.Config.DryMode` here? If not we could just keep the function, but use `option.Config.AreDevicesRequired()` internally.

Reply:
Yes, I don't think it really made sense to check that flag in the first place. I'll probably just send a PR to deprecate it.